package.json

```diff
@@ -18,15 +17,17 @@
     "dev": "vitest",
     "build": "tsup src/index.ts --format cjs,esm --dts",
     "lint": "tsc",
-    "release": "turbo build lint test && changeset publish"
+    "attw": "attw --pack",
+    "release": "turbo build lint test attw && changeset publish"
```

Comment:
You'll need to add attw to turbo.json to get this working I think.

What outputs does attw --pack have?

Reply:
Just an exit code (and stdout whether it succeeds or fails, but only interesting in CI if you need to look at why it failed). It runs `npm pack` but deletes the file after checking it.